### PR TITLE
chore: release

### PR DESCRIPTION
**Pull request by dhayab**
This pull request prepares the following release:
| Repository | Branch | Update |
|---|---|---|
| [instantsearch](https://github.com/algolia/instantsearch) | chore/release-1694529882358 | minor |

### Release Summary
This is going to be published with the following command:

```yarn lerna publish from-package --yes```

### Merging Instructions
When merging this pull request, you need to **Squash and merge** and make sure that the title starts with `chore: release`.
<details>
<summary>See details</summary>

After that, a commit `chore: release` will be added and you or your CI can run `shipjs trigger` to trigger the release based on the commit.
Squash and merge
</details>

---

_This pull request is automatically generated by [Ship.js](https://github.com/algolia/shipjs)_.

### CODE REVIEW

File: packages/instantsearch.css/CHANGELOG.md
Comment (dhayab) [inline packages/instantsearch.css/CHANGELOG.md:1]: This isn't right. I'll fix it manually.

File: specs/CHANGELOG.md
Comment (dhayab) [inline specs/CHANGELOG.md:1]: This isn't right. I'll fix it manually.

**Review by dhayab — COMMENTED** [review]
There are some whitespace correction due to shipjs working on top of linted files from last PR, and 2 packages have version bumps when they shouldn't (see below).

File: packages/instantsearch.css/CHANGELOG.md
Comment (Haroenv) [reply packages/instantsearch.css/CHANGELOG.md:1]: it seems right now though?

**Review by Haroenv — APPROVED** [review]
lgtm (although possibly a couple internal-only changes trigger a release. Fine for me

**Review by Haroenv — CHANGES_REQUESTED** [review]
added the insights PR #5848

File: packages/instantsearch.js/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 # Change Log
 
-All notable changes to this project will be documented in this file. See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [4.56.11](https://github.com/algolia/instantsearch/compare/instantsearch.js@4.56.10...instantsearch.js@4.56.11) (2023-09-12)
+
+**Note:** Version bump only for package instantsearch.js
```
Comment (dhayab) [inline packages/instantsearch.js/CHANGELOG.md:8]:
```suggestion
### Bug Fixes

- **insights**: use init method params type from Search Insights ([#5848](https://github.com/algolia/instantsearch/issues/5848)) ([d05c6c6](https://github.com/algolia/instantsearch/commit/d05c6c69c7662c9551431ae787223029076ff972))
```

**Review by Haroenv — APPROVED** [review]
lgtm (if you don't mind that we'll solve the vue templates later)

**Pull request merged**